adobe_vipm/flows/mpt.py

```diff
@@ -15,7 +15,7 @@ def _has_more_pages(page):
 @wrap_http_error
 def get_agreement(mpt_client, agreement_id):
     response = mpt_client.get(
-        f"/commerce/agreements/{agreement_id}?select=seller,buyer"
+        f"/commerce/agreements/{agreement_id}?select=seller,buyer,listing"
```

Comment:
Is it going to fail existing solution, while listings are not there?

Reply:
I will revert for the moment